services/frontend-service/pkg/handler/handleEnvironments.go

```diff
@@ -68,3 +68,20 @@ func (s Server) HandleEnvironments(w http.ResponseWriter, req *http.Request, tai
 		http.Error(w, fmt.Sprintf("unknown function '%s'", function), http.StatusNotFound)
 	}
 }
+
+func (s Server) handleApiEnvironments(w http.ResponseWriter, req *http.Request, tail string) {
+	environment, tail := xpath.Shift(tail)
+	if environment == "" {
+		http.Error(w, "missing environment ID", http.StatusNotFound)
+		return
+	}
+
+	function, tail := xpath.Shift(tail)
+
+	switch function {
+	case "releasetrain":
```

Comment:
please give a more detailed commit message / PR description.
It should at least mention what the API url looks like, and describe in one sentence what it does (maybe something like "it runs the same logic as a release train and then returns what would change")